lib/logstash/config/mixin.rb

```diff
@@ -121,13 +126,17 @@ def config_name(name=nil)
       return @config_name
     end
 
+    # Deprecated: Declare the version of the plugin
+    # inside the gemspec.
     def plugin_status(status=nil)
```

Comment:
nitpicking but we have space for all assignments, why not in method definitions?
```ruby
def plugin_status(status = nil)
```